Automattic/blaze-ads
# Updates Blaze module to latest version

### What and why? 🤔

I am updating the Blaze module to the latest version. I also updated other composer dependencies that were very old and deprecated.

### Testing Steps ✍️

* Install this version of the plugin and test that it works fine. The easiest way to test is to create a temporary site and then connect the plugin to WPCOM.

The new Blaze version will allow you to interact with the dashboard while the sync is in progress, and any new post will appear instantly in the list.

![Screenshot 2024-10-30 at 3 21 25 PM](https://github.com/user-attachments/assets/b14dd06c-858f-48e2-9728-7cfaf6d82e94)


### Review checklist
